packages/webpack-cli/lib/utils/GroupHelper.js

```diff
@@ -22,14 +22,18 @@ class GroupHelper {
     }
 
     arrayToObject(arr) {
-        if (!arr) {
-            return;
+        if (Array.isArray(arr)) {
```

Comment:
if(Array.isArray(arr) return 
if !arr return

Reply:
I have added  object validation in tests. So, to verify  that if it's an `object` or an `Array`, I added that blocks.

Reply:
That’s not what I’m saying

Reply:
Thanks

Reply:
I resolved it.